packages/bsky/src/db/tables/blob-takedown.ts

```diff
@@ -0,0 +1,9 @@
+export interface BlobTakedown {
```

Comment:
I don't think we're tracking blobs anywhere else in the appview, so needed a new table for takedown state